i18n/en.json

```diff
@@ -1921,6 +1921,7 @@
   "admin.userManagement.userDetail.username": "Username",
   "admin.viewArchivedChannelsHelpText": "(Beta) When true, allows users to view, share and search for content of channels that have been archived. Users can only view the content in channels of which they were a member before the channel was archived.",
   "admin.viewArchivedChannelsTitle": "Allow users to view archived channels:",
+  "admin.warn_metric.getting_trial": "Starting trial",
```

Comment:
@esadur is there a doc where I can review these messages and the decisions made to choose the specific framing?

Reply:
Admin Advisor - Text Copy. Sent to project channel in MM.